train.py

```diff
@@ -302,8 +306,13 @@ def get_lr(it):
                     'best_val_loss': best_val_loss,
                     'config': config,
                 }
+                out_dir = 'out'
                 print(f"saving checkpoint to {out_dir}")
-                torch.save(checkpoint, os.path.join(out_dir, 'ckpt.pt'))
+                weight_file = os.path.join(out_dir, f'ckpt_{iter_num}.pt')
+                torch.save(checkpoint, weight_file)
+                # Add a fork call to evaluation process
+                temperature = 0.01
```

Comment:
make this configurable (put at the top of the file before the configurator line)